clusters/build-clusters/01_cluster/openshift-machine-api/admin_build01-9hdwj-worker-us-east-1a-debug_machine_set.yaml

```diff
@@ -0,0 +1,66 @@
+apiVersion: machine.openshift.io/v1beta1
+kind: MachineSet
+metadata:
+  labels:
+    machine.openshift.io/cluster-api-cluster: build01-9hdwj
+  name: build01-9hdwj-worker-us-east-1a-debug
+  namespace: openshift-machine-api
+spec:
+  replicas: 1
```

Comment:
@hongkailiu I am unsure of this value, should it be set to 1? How do we specify how many worker nodes should be part of the debug pool as well?

Reply:
I have set the node-role label in the machine set https://github.com/openshift/release/pull/17307/files#diff-4e60cd39eaa70f90b2e2bdd7e591f978a4d459b29cd2d0dae460b10a4b1e4ed6R27, so it will automatically match this to the node-role label set in the MCP definition.
Looked through the docs and the machine set will create one new node with the debug node-role label and will be part of the custom debug pool. Since there is no taints on it, the ci workloads can be scheduled to this debug node as well.